frontend/src/scenes/pipeline/pipelineNodeNewLogic.tsx

```diff
@@ -42,11 +49,20 @@ export const pipelineNodeNewLogic = kea<pipelineNodeNewLogicType>([
                     path: urls.pipeline(stage ? NODE_STAGE_TO_PIPELINE_TAB[stage] : undefined),
                 },
                 {
-                    // TODO: use the plugin name
                     key: pluginId || batchDestination || 'Unknown',
-                    name: pluginId ? pluginId.toString() : batchDestination ?? 'Options',
+                    name: pluginId ? 'New' : batchDestination ? `New ${batchDestination} destination` : 'Options',
                 },
             ],
         ],
+        batchExportServiceNames: [
+            (s) => [s.user],
+            (user): BatchExportService['type'][] => {
+                const services: BatchExportService['type'][] = ['BigQuery', 'Postgres', 'Redshift', 'Snowflake', 'S3']
+                if (user?.is_impersonated) {
```

Comment:
What about `is_staff` (admin logged in normally as foo@posthog.com)?

Reply:
To add

Reply:
Unless we really don't want to ever access this internally

Reply:
we can always impersonate ourselves ;)